pom.xml

```diff
@@ -1411,6 +1411,12 @@
                     <artifactId>provisio-maven-plugin</artifactId>
                     <version>1.0.9</version>
                 </plugin>
+
+                <plugin>
+                    <groupId>pl.project13.maven</groupId>
+                    <artifactId>git-commit-id-plugin</artifactId>
+                    <version>4.0.0</version>
```

Comment:
The commit message is pretty pale.
Please mention why the downgrade and why the version 4.0.0.

Also, mention how the problem manifests so that we can retest in the future.

Reply:
Once we verify that this is the real solution for the problem, we need to downgrade this in airbase too.